### PR TITLE
TOB recommendations

### PR DESCRIPTION
**Pull request by jesusholoo**
# TOB Fix Log

22. Contract name duplication leaves codebase error-prone - Not fixed

### CODE REVIEW

**Review by krlosMata — DISMISSED** [review]

File: docs/developers/dev-guide.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ withdrawn by the governance under the tutelage of Aragon court.
 ## Adding New Tokens
 Hermez contains a list with all tokens supported. The following list includes some requirements on the token listing:
 - Tokens must be ERC20
-- Only the governance can add new tokens.
+- Governance can decide the fee cost of adding tokens, therefore can regulate the token listing.
```
Comment (jleebee) [inline docs/developers/dev-guide.md]: remove the comma and replace with "and" - fee cost of adding tokens and therefore can regulate the token listing. 
:)

**Review by jleebee — CHANGES_REQUESTED** [review]
just one edit needed :)

File: docs/developers/dev-guide.md (hunk 1 above)
Comment (jesusholoo) [reply docs/developers/dev-guide.md]: ty!!

**Review by jleebee — DISMISSED** [review]

**Review by jleebee — APPROVED** [review]

**Review by eduadiez — APPROVED** [review]
LGTM

**Pull request merged**